notes_app/client/templates/users_list.tpl.html

```diff
@@ -0,0 +1,25 @@
+<div>
+    <style>
+        .ui-grid-header-cell {
+            height: 30px;
+        }
+    </style>
+    <div
+            ui-grid="$ctrl.gridOptions"
+
```

Comment:
tooooooooooooooo maaaaaaaaaaaaaaaanyyyyyyyyyyyyy empty lines, it's harder to read. And it's a very big indentation - 4 spaces - in html.